pkgs/by-name/cl/clouddrive2/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+}:
+let
+  os = if stdenv.hostPlatform.isDarwin then "macos" else "linux";
+  arch = if stdenv.hostPlatform.isAarch64 then "aarch64" else "x86_64";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "clouddrive2";
+  version = "0.7.21";
+
+  src = fetchurl {
+    url = "https://github.com/cloud-fs/cloud-fs.github.io/releases/download/v${finalAttrs.version}/clouddrive-2-${os}-${arch}-${finalAttrs.version}.tgz";
+    hash =
+      {
+        x86_64-linux = "sha256-U8th7cEAFDBg+CV6DEX0YnqaM2+NJNKmyA/IRtnWESo=";
+        aarch64-linux = "sha256-LaAVO4p0sSfsGmFPSzly1Hzo+t4oY6mgaTdnHrhu0vI=";
+        x86_64-darwin = "sha256-/cyLcX7A+WpuS6yciqAz3jvDIJvXEoyQyX+xe6+eRmE=";
+        aarch64-darwin = "sha256-Y5cKIHVzX0TjMNZGW3YvU8MNpuGsIjp4qNlB+b2BRJM=";
+      }
+      .${stdenv.hostPlatform.system} or (throw "unsupported system ${stdenv.hostPlatform.system}");
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    mkdir -p $out/opt/clouddrive2
+    cp -r wwwroot "$out/opt/clouddrive2/wwwroot"
+    cp -r clouddrive "$out/opt/clouddrive2/clouddrive"
+    makeWrapper $out/opt/clouddrive2/clouddrive $out/bin/clouddrive
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = ./update.sh;
```

```suggestion
  '';
  
  nativeInstallCheckPhaseInputs = [
    versionCheckHook
  ];
  checkVersionProgramArg = [ "--version" ];
  doInstallCheck = true;

  passthru.updateScript = ./update.sh;
```

Can you check if this works please ?

---

It should seem to be `versionCheckProgramArg` here instead.